### ISSUE TITLE
examples of TabTemplate

### ISSUE THREAD

**Issue by jpage4500** [android/car-samples#43]
I see examples of TabTemplate in the design guide ([here](https://developer.android.com/static/training/cars/Android%20for%20Cars%20App%20Library%20design%20guidelines.pdf))

Can someone add some examples of this template to the car-samples project?

<img width="1145" alt="image" src="https://user-images.githubusercontent.com/1895553/235218745-f7341ae9-6ab2-4d14-a301-775a5178194a.png">

**Comment by dshokouhi** [comment]
i agree some examples would be great, trying to implement this and hard to decipher what needs to be done when looking at the docs

**Comment by vincentstocks** [comment]
Has anyone found any examples? I just cannot get it to work correctly.
